playbooks/aws/openshift-cluster/tasks/launch_instances.yml

```diff
@@ -2,8 +2,8 @@
 - set_fact:
     created_by: "{{ lookup('env', 'LOGNAME')|default(cluster, true) }}"
     docker_vol_ephemeral: "{{ lookup('env', 'os_docker_vol_ephemeral') | default(false, true) }}"
-    env: "{{ cluster }}"
-    env_host_type: "{{ cluster }}-openshift-{{ type }}"
+    cluster: "{{ cluster }}"
+    cluster_host_type: "{{ cluster }}-openshift-{{ type }}"
```

Comment:
do we need cluster_host_type still? shouldn't tooling query for the separate tags?